tests/queries/0_stateless/02911_backup_restore_keeper_map.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+database_name="$CLICKHOUSE_DATABASE"_02911_keeper_map
+$CLICKHOUSE_CLIENT -nm -q "
+    DROP DATABASE IF EXISTS $database_name;
+    CREATE DATABASE $database_name;
+    CREATE TABLE $database_name.02911_backup_restore_keeper_map1 (key UInt64, value String) Engine=KeeperMap('/' || currentDatabase() || '/test02911') PRIMARY KEY key;
+    CREATE TABLE $database_name.02911_backup_restore_keeper_map2 (key UInt64, value String) Engine=KeeperMap('/' || currentDatabase() || '/test02911') PRIMARY KEY key;
+    CREATE TABLE $database_name.02911_backup_restore_keeper_map3 (key UInt64, value String) Engine=KeeperMap('/' || currentDatabase() || '/test02911_different') PRIMARY KEY key;
+
```

Comment:
It's difficult to understand that keeper_map1 and keeper_map2 are actually one table. Please add a comment about that.